Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -693,24 +945,36 @@ private void AddCorners()
                     GameObject cornerVisuals = Instantiate(isFlattened ? scaleHandleSlatePrefab : scaleHandlePrefab, visualsScale.transform);
                     cornerVisuals.name = "visuals";
 
-                    if(isFlattened)
+                    // this is the size of the corner visuals
+                    var cornerbounds = GetMaxBounds(cornerVisuals);
+                    // we need to multiply by this amount to get to desired scale handle size
+                    var invScale = scaleHandleSize / cornerbounds.size.x;
+                    cornerVisuals.transform.localScale = new Vector3(invScale, invScale, invScale);
+
+                    if (isFlattened)
                     {
                         // Rotate 2D slate handle asset for proper orientation
                         cornerVisuals.transform.Rotate(0, 0, -90);
                     }
 
                     ApplyMaterialToAllRenderers(cornerVisuals, handleMaterial);
 
-                    if (hideElementsInInspector == true)
-                    {
-                        corner.hideFlags = HideFlags.HideInHierarchy | HideFlags.HideInInspector;
-                    }
 
                     corners.Add(corner.transform);
                 }
             }
         }
 
+        private Bounds GetMaxBounds(GameObject g)
+        {
+            var b = new Bounds(g.transform.position, Vector3.zero);
```

Comment:
Can't assume the center of the object is necessarily part of its geometry.

Reply:
Good call.